Document unresponsive menu activation (#14)
VSpaceCode/vspacecode.github.io: docs/troubleshooting.md

],
```

## Unresponsive menu activation

> The input box of which-key menu for key input is not displayed some delay after the `<spc>` is pressed on a buffer. This is likly due to conflicting vim binding that starts with `<spc>`. For example, the vim binding of `[" ", "d"]` in the following `settings.json` example is causing the delay becuase vim is waiting for the second input  when `<spc>` is pressed.

Reviewer: ```suggestion
If you press `<spc>` on a buffer and the which-key menu doesn't appear immediately, you might have one of the following problems.

### Conflicting vim binding

If in your `settings.json` file you have a vim binding that starts with `<spc>`, vim will wait for the second input when `<spc>` is pressed.

Example:
```

```json
"vim.normalModeKeyBindingsNonRecursive": [
  {
    "before": [" ", "d"],
    "after": ["d", "d"]
  },
  {
    "before": ["<space>"],
    "commands": ["vspacecode.space"]
  }
]
```

> You can remove the conflicting vim bindings from your `settings.json` completely, or use the VSpaceCode's [overrides](./menu-customization#addreplace) instead

Reviewer: ```suggestion

In order to solve it, remove the conflicting vim bindings from your `settings.json` completely, or use the VSpaceCode's [overrides](./menu-customization#addreplace) instead.
```

> 

Reviewer: ```suggestion
### Virtual Machine or slow hardware

If you are working on limiting resources consider using [VSCode remote](https://code.visualstudio.com/docs/remote/remote-overview)

### Other conflicts

Try to remove all the extensions except the ones installed by VSpaceCode, and clean your `settings.json` and `keybindings.json` files in order to spot some weird conflicts.
```
